workflows/rules/qc/coverage.smk

```diff
@@ -14,7 +14,7 @@ rule wgs_coverage:
     output:
         "{workingdir}/{stype}/reports/{sname}_WGScov.tsv"
     shell:
-        "{params.sentieon} driver -i {input} -r {params.reference} "
+        "{params.sentieon} driver -t {params.threads} -i {input} -r {params.reference} "
```

Comment:
This rule is rarely used (it is for hg19). coverage_hg38.smk is the rule for hg38, should we add params.threads in this rule as well?

Reply:
If it's missing then yes, seems relevant.

Reply:
I'll add it in the current branch I'm working on